docs/rules/html-attributes-casing.md

```diff
@@ -0,0 +1,35 @@
+# Define a style for the attributes casing in templates. (html-attributes-casing)
```

Comment:
I like it:
1. change name of rule to attribute-hyphenation
2. apply rule only on components `isCustomComponent`
3. change options to `always` (default) or `never`
4. add `aria-` to exceptions